docs/development/debugging/index.md

```diff
@@ -1,37 +0,0 @@
-# Debugging
-
-Deploy REANA in debug mode:
-
-**1.** Create the cluster with the REANA source code mounted:
-
-```console
-$ reana-dev cluster-create --mode=debug
-```
-
-**2.** Build the component you want to debug with debugging dependencies:
-
-```console
-$ reana-dev docker-build -b DEBUG=1 -c reana-server
```

Comment:
Maybe here we can add here `reana-dev kind-load-docker-image -c reana-server` as we expect developers to use `kind`.
_(Thinking about the wiki page...)_

Reply:
I've just created a debugging wiki page [here](https://github.com/reanahub/reana/wiki/Debugging) and added your suggested line. Please have a look if everything is ok there.

Reply:
Looks good, but the screenshot is not visible.

Reply:
Nice catch, just brought back the screenshot to the wiki page. But the question now is where is the proper place to store it. The screenshot is located in this repo, but used in `reana` wiki. Probably we should move it there? @tiborsimko